packages/oas/test/analyzer/queries/openapi.test.ts

```diff
@@ -89,8 +89,13 @@ describe('analyzer queries (OpenAPI)', () => {
   describe('circularRefs', () => {
     it('should determine if a definition has circular refs when it does', async () => {
       await expect(QUERIES.circularRefs(circular)).resolves.toStrictEqual([
+        '#/components/schemas/BodyPart/properties/parent',
```

Comment:
`json-schema-ref-parser` apparently got some improvements to circular reference detection because these new expectations are definitely circular.